Makefile.in

```diff
@@ -387,6 +393,9 @@ asynclook$(EXEEXT):	$(ASYNCLOOK_OBJ_LINK) libunbound.la
 streamtcp$(EXEEXT):	$(STREAMTCP_OBJ_LINK)
 	$(LINK) -o $@ $(STREAMTCP_OBJ_LINK) $(SSLLIB) $(LIBS) $(DYNLIBMOD_EXTRALIBS)
 
+dohclient$(EXEEXT):	$(DOHCLIENT_OBJ_LINK)
+	$(LINK) -o $@ $(DOHCLIENT_OBJ_LINK) $(SSLLIB) $(LIBS)
```

Comment:
If streamtcp needs $(DYNLIBMOD_EXTRALIBS) then also this one needs it added after $(LIBS) on the link line.  That dynlibmod was added after the DOH work started, but it'd be good to fix.